scripts/delete-unused-users.py

```diff
@@ -36,15 +41,30 @@ async def main():
                 except:
                     print(user['last_activity'])
                     raise
-                if last_activity and datetime.now().astimezone() - last_activity < timedelta(hours=24) or user['server'] is not None:
+                last_activity = asyncio
```

Comment:
I'm unclear on this since 1) it overrides the assignment in the `try` above and it will never represent the user's last activity, and 2) `asyncio` is a library.